plugins/com.google.cloud.tools.eclipse.dataflow.ui/plugin.xml

```diff
@@ -227,21 +227,21 @@
          </separator>
          <command commandId="com.google.cloud.tools.eclipse.dataflow.ui.dataflowVersion.pin"
              label="Pin Dataflow Version"
-             tooltip="Pin the Dataflow version for this project to the current latest version">
+             tooltip="Pin the Dataflow Version for this Project to the Current Latest Version">
            <visibleWhen>
              <reference definitionId="com.google.cloud.tools.eclipse.dataflow.ui.tracksDataflowVersion" />
            </visibleWhen>
          </command>
          <command commandId="com.google.cloud.tools.eclipse.dataflow.ui.dataflowVersion.track"
              label="Track Dataflow Version"
-             tooltip="Tracks the Latest Dataflow version in this project">
+             tooltip="Tracks the Latest Dataflow Version in this Project">
```

Comment:
It's a definite article here, not a pronoun: http://partofspeech.org/what-part-of-speech-is-this/